ldapdb/backends/ldap/base.py

```diff
@@ -208,7 +210,12 @@ def get_new_connection(self, conn_params):
 
         options = conn_params['options']
         for opt, value in options.items():
-            connection.set_option(opt, value)
+            if opt.upper() == 'TIMEOUT':
+                connection.timeout = int(value)
+            elif opt.upper() == 'BATCH_SIZE':
```

Comment:
You're right, I have to fix this, Django uses lower snake case for the `CONNECTION_OPTIONS` keys.